feature-group-definitions/scroll-driven-animations.yml

```
@@ -1,4 +1,5 @@
 name: Scroll-driven animations
+description: CSS scroll-driven animations are a type of CSS animations that don't run over time, but instead are driven by the user's scroll position through the page.
```

I think it applies to all scroll containers, so dropping "through the page":

```suggestion
description: CSS scroll-driven animations are a type of CSS animations that don't run over time, but are instead driven by the user's scroll position.
```